lib/streamlit/file_util.py

```diff
@@ -137,3 +137,34 @@ def file_is_in_folder_glob(filepath, folderpath_glob):
 
     file_dir = os.path.dirname(filepath) + "/"
     return fnmatch.fnmatch(file_dir, folderpath_glob)
+
+
+def file_in_pythonpath(filepath):
+    """Test whether a filepath is in the same folder of a path specified in the PYTHONPATH env variable.
+
+
+    Parameters
+    ----------
+    filepath : str
+        An absolute file path.
+
+    Returns
+    -------
+    boolean
+        True if contained in PYTHONPATH, False otherwise. False if PYTHONPATH is not defined or empty.
+
+    """
+
+    if "PYTHONPATH" not in os.environ:
+        return False
+    pythonpath = os.environ["PYTHONPATH"]
+    if len(pythonpath) == 0:
+        return False
```

Comment:
Nit: I think this can be shortened a bit to:
```python
pythonpath = os.environ.get("PYTHONPATH", "")
if len(pythonpath) == 0:
    return False
```

Reply:
Like this!